README.md

```diff
@@ -101,6 +101,24 @@ https://docs.docker.com/engine/install/linux-postinstall/)
     ./build.sh
     ```
 
+## Running Smoke Tests on the Kubernetes Cluster
```

Comment:
Thanks for the instructions. I think this should be a level up in headers since it's not really a subsection of `Build and push a new container`.

I'm wondering if we should add a final step to `How to release a new version of BraveSensor-Server` to run the smoke tests. Do you feel confident that this could run safely in production?

Reply:
I am pretty confident yeah, but was envisioning this as something that happens in staging, so to speak, before you upgrade the production development. So it would actually be more a pre-step than a final one?

Reply:
That sounds good to me too. Is the staging environment ready to be used?

Reply:
it's ready, it could use some enhancements like its own redis instance and its own database, but I've been using it to test this already without issues

Reply:
Then that makes sense. A pre-final step to deploy to staging and run the smoke tests there seems like a great idea.

Reply:
I also think this makes sense!